repository.json

```diff
@@ -175,6 +175,16 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/LDAP/LSP-ltex-ls",
+			"name": "LSP-ltex-ls",
+			"releases": [
+				{
+					"sublime_text": ">=3154",
```

Comment:
Your plugin only supports ST4.

(BTW. We are a little hesitant to add packages that only work on ST4 since it's not even public yet)

Reply:
Fixed. I do understand that :)